docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,18 @@
+# Contributing to `nufmt`
+
+Salutations! Thanks for coming by and the interest into this project!
+We would like to order the contributions like this:
+
+- Before you start hacking, it is important to **ask first** if your idea or bugfix is in order. Create an issue or [come and say hi in the discord](https://discord.gg/NtAbbGn). We don't bite!.
+
+  It would be sad that you do the effort to clone the project, successfully make the PR, but it wasn't in our plans or there is another PR that is currently adressing that issue.
+
+- After the PR is submitted, the workflows will start to lint, check and test the changes. Please try to stay all green ✅.
+- Sometimes we can take some time to respond. Sorry, we are few and there is much to do here!
+
+## General guidelines and philosophy
+
+This is a list of things we would like to have and mantain across time. Please do your best to abide by.
+
+- Everything should be explained: rust docs, comments, drawings, pick what makes you comfortable, but it is important to make it clear. There will always be some new guy or gal into the project we want to welcome 😄.
```

Comment:
agree with these guidelines, i'm just not sold on comments because they most of the time can be refactored with
- functions
- better variables names

:thinking:

Reply:
thanks @AucaCoyan :blush: